app/views/content/funding-and-support/if-you-come-from-outside-england.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "If you come from outside England"
+heading: "If you come from outside England"
+description: |-
+  Find out about the different arrangements for getting financial support if you live outside England.
+related_content:
+    Your initial teacher training year: "/train-to-be-a-teacher/initial-teacher-training"
+    Get school experience: "/train-to-be-a-teacher/get-school-experience"
+    5 reasons to attend a Train to Teach event: "/blog/5-reasons-to-attend-a-train-to-teach-event"
+promo_content:
+    - content/funding-and-support/promos/get-adviser-support-promo
```

Comment:
@gemmadallmandfe wherever we have an adviser promo on a page, there's the adviser CTA in the footer immediately below it. Do we need to review the adviser CTA in the footer/the adviser promo?